winpkg/ddev.nsi

```diff
@@ -15,9 +15,9 @@ CRCCheck On
 !include LogicLib.nsh
 
 ; The name of the installer
-Name "ddev ${VERSION}"
+Name "ddev ${MUI_VERSION}"
```

Comment:
This is not necessary but IMO makes sense to use only the previously defined constant